internal/cephfs/mounter/fuse.go

```diff
@@ -47,7 +47,10 @@ var (
 
 type FuseMounter struct{}
 
-func mountFuse(ctx context.Context, mountPoint string, cr *util.Credentials, volOptions *store.VolumeOptions) error {
+func mountFuse(ctx context.Context,
```

Comment:
I am looking at this, and wonder what changed in addition to the formatting? If it is really formatting only, please put that in a different commit or PR.

Same counts for the function signature updates elsewhere.

Reply:
It was leftover from my previous changes, removed now